### DISCUSSION TITLE
Additional Tagging for Frigate Event

### DISCUSSION THREAD

**Discussion by rjdainty1** [blakeblackshear/frigate#3756] [Ask A Question] [answered]
Hello,

I wondered if it were possible to update / tag an event with specific values, ideally via an API call or MQTT event.  On certain Frigate events, I process the image with further detection and generate a result string.  I'd like to append this result string to the original event in Frigate.  

Alternatively I need to keep a separate log of these events with my externally generated tags / strings which seems a shame given I have the majority of information already in Frigate.

Is anyone aware of a way to update the Frigate event?

Thanks

**Comment by NickM-27** [comment, marked as answer]
We have sub labels in frigate 0.11 and I added an http API to that as well, check out the RC if you haven't already.

**Reply by rjdainty1** [reply]
That's excellent, thanks.  Works a treat.
